keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -287,6 +367,11 @@ Pod with ResizePolicy set to NoRestart for all its Containers.
      - Evict the Pod to trigger a replacement Pod with new desired resources,
      - Do nothing and let Kubelet back off and later retry the in-place resize.
 
+<<[UNRESOLVED]>>
+I'm suggesting that there be a real signal indicating "I tried to resize, but I
+can't" beyond the lack of update to `allocated`.
```

Comment:
I tried - https://github.com/kubernetes/enhancements/pull/686#discussion_r297384981

Reply:
See my argument above - lack of signal is a poor signal, IMO